### PR TITLE
accountsservice: update to 0.6.53.

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

**Comment by Cogitri** [comment]
Breaks login on at least musl as of now...

**Comment by Cogitri** [comment]
Ah, it's just GNOME stuff not finding the typelib of accountsservice (it's `accountsservice-1.0.typelib` instead of `AccountSservice-1.0.typelib` now)

File: srcpkgs/accountsservice/template [hunk 1]
```diff
@@ -24,20 +21,19 @@ build_options="gir"
...
-			# FIXME: if ac_cv_header_utmpx_h=no is set, the function
-			# wtmp_helper_update_login_frequencies() is not defined.
-			# configure_args+=" ac_cv_header_utmpx_h=no"
-			;;
-	esac
-}
+case "$XBPS_TARGET_MACHINE" in
+	*-musl)
+		pre_configure() {
+			patch -Np0 -i "${FILESDIR}"/musl-wtmp.patch
```
Comment (maxice8) [inline srcpkgs/accountsservice/template]: Does it really need to be only on musl ?

Comment (Cogitri) [reply srcpkgs/accountsservice/template]: Ah, no, glibc shouldn't even reach the `elif` statement the patch changes

File: srcpkgs/accountsservice/template [hunk 2]
```diff
@@ -24,21 +21,11 @@ build_options="gir"
 # Disable gir for cross builds.
 if [ -z "$CROSS_BUILD" ]; then
 	build_options_default="gir"
+else
+	# for ITS rules
+	hostmakedepends+=" polkit-devel"
```
Comment (maxice8) [inline srcpkgs/accountsservice/template]: only polkit is required

```
# xbps-query -Rf polkit | grep its
/usr/share/gettext/its/polkit.its
/usr/share/gettext/its/polkit.loc
```

**Comment by Cogitri** [comment]
.53 is .52 but with my g-i patch upstreamed.

**Pull request merged**